testplan/common/utils/path.py

```diff
@@ -51,21 +51,21 @@ def workspace_root():
     return pwd()
 
 
-def default_runpath(entity):
+def default_runpath(entity=None):
```

Comment:
i think this default_runpath is only meant to be used to decide the default runpath for TestRunner. For other entities, the right logic to determine the runpath, shall at least take a look at its parent. I would prefer NOT to change this API for our test code usage.

Reply:
recovered